e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -218,6 +218,74 @@ describe('Desktop notifications', () => {
             });
         });
     });
+
+    it('MM-T488 Desktop Notifications - Teammate name display set to username', () => {
```

Comment:
Wasn't this covered by https://github.com/mattermost/mattermost-webapp/pull/7154 already?

Reply:
i totally forgot to change the base branch, this PR should point to https://github.com/mattermost/mattermost-webapp/pull/7154 since is branched off from there. Updating now